docs/reference-architectures/hybrid-networking/vnet-topology-hub-spoke.md

```diff
@@ -0,0 +1,430 @@
+---
+title: Implementing a hub-spoke topology in Azure
+description: >-
+  How to implement a hub-spoke network topoly in Azure.
+author: telmosampaio
+ms.service: guidance
+ms.topic: article
+ms.date: 05/05/2017
+ms.author: pnp
+
+pnp.series.title: Implement a hub-spoke network topology in Azure
+pnp.series.prev: expressroute
+cardTitle: Improving availability
+---
+# Implement a hub-spoke topology in Azure
+
+This reference architecture shows how to implement a hub-spoke topology in Azure, using the hub as a gateway between Azure and your on-premises datacenter.
+
+Traffic flows between the on-premises datacenter and Azure through an ExpressRoute or VPN gateway connection. This connection is made to a hub virtual network (VNet) in Azure, which in turn is peered to other VNets in the same Azure region, as seen in the piture below.[**Deploy this solution**](#deploy-the-solution).
+
+> [!NOTE]
+> Azure has two different deployment models: [Resource Manager](/azure/azure-resource-manager/resource-group-overview) and classic. This reference architecture uses Resource Manager, which Microsoft recommends for new deployments.
+
+Typical uses for this architecture include:
+
+* Workloads deployed in different environments (development, testing, production) that require the specific services that can be shared (in the hub VNet) while maintaining isolation (deployed to spokes).
```

Comment:
I don't understand this phrase: "that require the specific services that can be shared" - which services?